webknossos-datastore/app/com/scalableminds/webknossos/datastore/datavault/NullDataVault.scala

```diff
@@ -0,0 +1,8 @@
+package com.scalableminds.webknossos.datastore.datavault
+
+class NullDataVault extends DataVault {
```

Comment:
I see that this is used for the FileSystemVaultPath – would it make sense to name it FileSystemDataVault? If so, the `get` implementation using `Files.newByteChannel` could also live here

Reply:
Actually that does not work that well, since the readBytes implementation in FileSystemDataVault would still need a real nio Path, but the interface provides a VaultPath, which does not allow it to call Files.readAllBytes(path) because of the missing filesystem

Reply:
I see, then let’s consider this again later. I think, eventually there should be no more nio Paths used in the same places as VaultPaths. I created the follow-up issue https://github.com/scalableminds/webknossos/issues/6929